integreat_cms/cms/templates/pois/poi_list_archived_row.html

```diff
@@ -3,11 +3,13 @@
 {% load content_filters %}
 <tr class="border-t border-gray-200 hover:bg-gray-100 text-gray-800">
     <td class="py-3 pl-4 pr-2">
-        <input type="checkbox"
-               name="selected_ids[]"
-               value="{{ poi.id }}"
-               form="bulk-action-form"
-               class="bulk-select-item" />
+        {% if perms.cms.change_poi %}
```

Comment:
This has a small side effect: The rows loses the padding to the top and bottom, and shrinks a little bit (s. screenshot).

![Screenshot from 2024-12-17 16-23-12](https://github.com/user-attachments/assets/d5f4624e-2c65-44d4-85b1-3fa97fb773b5)

I'm sure there are better ways to do it, but I couldn't come up with one and wanted to share at least one idea, so here we go :)

I think I would move the permission check one hierarchy up.


```suggestion
               {% if perms.cms.change_poi %}
                        <td class="py-3 pl-4 pr-2">
```

and then add to next element the classes in case you can't change the poi (s. below).

Reply:
@JoeyStk thank you for your suggestion :) Unfortunately, if I put it above, the whole row moves on the left side, so I used instead the class option

Reply:
I somehow can't reproduce how it looks in the picture of @JoeyStk (Firefox/Chrome)

Reply:
Now it looks different and fine for me as well :) I think this is ready to be merged now :)